components/core/tools/docker-images/clp-env-base-ubuntu-jammy/Dockerfile

```
@@ -7,6 +7,9 @@ ADD ./tools/scripts/lib_install ./tools/scripts/lib_install
 
 RUN ./tools/scripts/lib_install/ubuntu-jammy/install-all.sh
 
+# Install uv
+RUN pip install uv
```

Hadolint (2.12.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)